networkx/algorithms/triads.py

```diff
@@ -224,6 +248,16 @@ def is_triad(G):
     -------
     istriad : boolean
        Whether G is a valid triad
+
+    Examples
+    --------
+    >>> G = nx.DiGraph([(1, 2), (2, 3), (3, 1)])
+    >>> nx.is_triad(G)
+    True
+    >>> G.add_edges_from([(0,1)])
```

Comment:
Just another nit - the code formatting needs to be updated here. Note that this should be done automatically for you if you have the latest [pre-commit hooks installed](https://networkx.org/documentation/latest/developer/contribute.html#development-workflow)

Reply:
I think the ```black``` linter is not working for the code in docstring. I had manually changed for the others previously. This was unfortunately left out. I will edit it.